app/views/main.scala.html

```diff
@@ -0,0 +1,25 @@
+@*
```

Comment:
Can you remove files and related code that just only obfuscating this service?

Reply:
I should have done that but I was occupied with other things.